Doc/whatsnew/3.10.rst

```diff
@@ -145,6 +145,14 @@ Add :data:`sys.orig_argv` attribute: the list of the original command line
 arguments passed to the Python executable.
 (Contributed by Victor Stinner in :issue:`23427`.)
 
+types
+-----
+
+Reintroduced the :data:`types.EllipsisType`, :data:`types.NoneType`
+and :data:`types.NotImplementedType` classes, providing a new set
+of types readily interpretable by type checkers.
```

Comment:
I think this and the blurb are fine, providing a reason without saying either that this is the only reason or that these should be use everywhere.